src/XRP/xrp-client.ts

```diff
@@ -139,4 +140,16 @@ export default class XrpClient implements XrpClientInterface {
   ): Promise<XrpTransaction | undefined> {
     return this.decoratedClient.getPayment(transactionHash)
   }
+
+  /**
```

Comment:
A pattern I really admired from Go is the ability to define an interface once and require it by other types. I notice that xrp-client.ts, xrp-client-interface, and xrp-client-decorator share the same API. Are there ways to only comment a single time?

Reply:
Not that I know of, *BUT* I am not a typescript expert. 

We've opted to over document rather than under document. At some point we'd like to generate docs from these comments and I assume that that point will be when we'll investigate the "right" way to do this. 

Open to feedback if you know the correct answer.